### ISSUE TITLE
Typo fixes and change PYTHONPATH editing instructions slightly

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4785]
|<img alt="joshg's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| joshg reported|
|-|-|
|Trac ID|trac#4785|
|Type|enhancement|
|Created|2010-12-31 21:25:28Z|

There was a typo: Systems Properites was mispelled.

Also, the first part of the example didn't match with the last part.  It was saying if your code is in "/tmp", then add "/my/stuff" to your PYTHONPATH.  Obviously, those things should match.

Also, the Windows paths were listed with forward slashes, which might work, but it's not very Windows.

Attachments:

* [typo-quotes-fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/dae/dae40cb6316d320130e860e2dd9e9177809d689e/17cdafa810b26f4864a734fbd1b87f2a6fc1ebe0.patch) (1350 bytes) - added by joshg on 2010-12-31 21:26:31Z - Fix typos and instructions

<details><summary>Searchable metadata</summary>

```
trac-id__4785 4785
type__enhancement enhancement
reporter__joshg joshg
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1293830728000000 1293830728000000
changetime__1293889883000000 1293889883000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks!  Looks great, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [30407]) Apply typo-quotes-fix.patch, fixing some confusion in the Quotes howto. 

Author: joshg
Reviewer: exarkun
Fixes: [#4785](https://github.com/twisted/twisted/issues/4785)

Clarify the PYTHONPATH setup instructions by using consistent path names and spelling
words correctly.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
